### PR TITLE
Add Code of Conduct

### PR DESCRIPTION
**Pull request by stefanv**
This is SciPy's code of conduct.

TODO: Decide who should serve on the CoC committee.

### CODE REVIEW

File: doc/source/conduct/code_of_conduct.rst [hunk 1]
```diff
@@ -0,0 +1,169 @@
+scikit-image Code of Conduct
+============================
+
+scikit-image adopts the `SciPy code of conduct
+<https://github.com/scipy/scipy/blob/master/doc/source/dev/conduct/code_of_conduct.rst>`__.
```
Comment (jni) [inline doc/source/conduct/code_of_conduct.rst]: Should this be a single underscore? From http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks : 

```rest
Use  `Link text <https://domain.invalid/>`_ for inline web links.
```

Comment (jni) [reply doc/source/conduct/code_of_conduct.rst]: Same for other links below.

File: doc/source/skips/1-governance.rst [hunk 2]
```diff
@@ -111,6 +111,8 @@ long-term, continued commitment to the project and its :ref:`values`. A
 nomination will result in discussion that cannot take more than a month and
 then admission to the SC by consensus.
 
+The scikit-image steering council may be contacted at
+[skimage-steering@groups.io](mailto:skimage-steering@groups.io).
```
Comment (jni) [inline doc/source/skips/1-governance.rst]: This is markdown in a ReST document.

**Comment by jni** [comment]
Thanks @stefanv! The build errors are probably related to the syntax errors I pointed out above. Re committee, I'd be happy to serve on it. I wonder if @emmanuelle and @willingc might be happy to join me? I hate to ask extra work of them but I'm reasonably confident that this will be a very low-volume list indeed. :grimacing:

File: doc/source/conduct/code_of_conduct.rst (hunk 1 above)
Comment (stefanv) [reply doc/source/conduct/code_of_conduct.rst]: https://necromuralist.github.io/posts/restructured-text-anonymous-links/

**Comment by stefanv** [comment]
Yes, that's very much the goal!  I am happy to serve on it too.

File: doc/source/skips/1-governance.rst (hunk 2 above)
Comment (NelleV) [reply doc/source/skips/1-governance.rst]: :rofl:

File: doc/source/conduct/code_of_conduct.rst (hunk 1 above)
Comment (emmanuelle) [reply doc/source/conduct/code_of_conduct.rst]: thanks for the link, it didn't know this either

**Comment by emmanuelle** [comment]
sure, I'd be happy to be in.

File: doc/source/skips/1-governance.rst (hunk 2 above)
Comment (emmanuelle) [reply doc/source/skips/1-governance.rst]: on a related note, did you know that one can build sphinx doc from a mixture of `rest` and `md` files? I find out this recently, including cross-links to pages. ReST is superior IMO for references (unfortunately no `:func:` in Markdown :-() but if we need to include a Markdown file at some point this is possible (with the `recommonmark` extension)

Comment (jni) [reply doc/source/skips/1-governance.rst]: @emmanuelle very interesting! Do you have a link to an example configuration/implementation?

**Comment by stefanv** [comment]
This is using ReCommonMark, but I have had mixed experience with it. Works 
fine for most Markdown, but fails on some corner cases (such as linking to 
online .md documents!). So, I prefer to use a dedicated tool. Pandoc to RST 
may be a more trustworthy pipeline.

**Comment by stefanv** [comment]
@hmaarrfk CoCs are not easy to write, and coming up with a version much shorter than this is hard.  You need to discuss the environment you want to operate in, then give examples of poor behavior, and finally address follow-up action.

For us, there is great benefit to adopting the SciPy CoC as-is.  Unless you have strong reason to modify theirs, I recommend we not do so. Changes should also preferably happen on SciPy instead.  Here are my comments on your responses, though.

>  Primarily in English??

I don't think we've ever done any official communication in a different language, except among local groups while working offline.

> I guess I understand what you are trying to say. But maybe just let people self filter until there is an over reporting problem???

I'm not sure I understand. This seems to me to make people aware that they could handle things informally, if that is the most appropriate mechanism.

> As reported by a reporter or as deemed by q member of the council? If I accuse JNI of something in this list, will JNI automatically loose affiliate status?

No, the reporter cannot decide when another person gets disconnected from the project.

File: doc/source/conduct/code_of_conduct.rst [hunk 3]
```diff
@@ -114,9 +114,8 @@ advice, in confidence.
 You can report issues to the scikit-image Code of Conduct committee, at
 skimage-conduct@groups.io. Currently, the committee consists of:
 
--  (chair)
-- 
-- 
+- Emmanuelle Gouillart <emmanuelle.gouillard@normalesup.org> (chair)
+- Juan-Nunez Iglesias <jni@fastmail.com>
```
Comment (jni) [inline doc/source/conduct/code_of_conduct.rst]: slight error in placement of hyphen... ;)

**Comment by jni** [comment]
@stefanv the committee quite clearly cannot consist of just you, me, and @emmanuelle. If you are eager to serve on it then you should replace me. =) The third person should be someone not close to us. It's always hard to find such people though. (We are having the same issue at napari.)

**Comment by sciunto** [comment]
I'm ready to be part of this committee if it helps.

**Comment by willingc** [comment]
@jni I'm willing to serve if needed.

**Comment by soupault** [comment]
I would also be happy to join.

**Comment by jni** [comment]
@stefanv I propose that you add @willingc (:wave:) and @soupault and then we can merge this! (assuming we can get CI to pass...!)

**Comment by sciunto** [comment]
Can I know why you rejected my application @jni (even in private)?

**Comment by stefanv** [comment]
@sciunto Thank you for volunteering; I suspect @jni missed your comment.

@jni I did not add myself to the list, as you can see from my last PR; I think we have enough other good volunteers.

How about we do @willingc, @emmanuelle, @sciunto, and @soupault?

**Comment by jni** [comment]
@sciunto no important reason!!! I was vaguely maximising for people who don’t know each other, and both you and I know Emma pretty well. I’m super happy to take @stefanv’s suggestion and put you in instead of me. I am just wary of asking people for work, since I am the only one actually paid to work here. =)

**Comment by stefanv** [comment]
@sciunto @willingc @soupault  Is it OK to add your emails next to your names?  If so, which do you prefer?

Thank you for your service!

**Comment by soupault** [comment]
@stefanv no objections :). The one from my github profile will do.

File: doc/source/conduct/code_of_conduct.rst [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+(see below). For possibly unintentional breaches, you may reply to the person
+and point out this code of conduct (either in public or in private, whatever is
+most appropriate). If you would prefer not to do that, please feel free to
+report to the Code of Conduct Committee directly, or ask the Committee for
+advice, in confidence.
+
+You can report issues to the scikit-image Code of Conduct committee, at
+skimage-conduct@groups.io. Currently, the committee consists of:
+
+- Emmanuelle Gouillart <emmanuelle.gouillard@normalesup.org> (chair)
```
Comment (sciunto) [inline doc/source/conduct/code_of_conduct.rst]:
```suggestion
- Emmanuelle Gouillart <emmanuelle.gouillard@normalesup.org> (chair)
- François Boulogne <devel@sciunto.org>
```

File: doc/source/conduct/code_of_conduct.rst [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+- Juan-Nunez Iglesias <jni@fastmail.com>
```
Comment (willingc) [inline doc/source/conduct/code_of_conduct.rst]:
```suggestion
- Juan-Nunez Iglesias <jni@fastmail.com>
- Carol Willing <willingc@gmail.com>
```

Comment (willingc) [reply doc/source/conduct/code_of_conduct.rst]: @stefanv Added my contact as a suggested change.

File: doc/source/conduct/code_of_conduct.rst [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+
```
Comment (soupault) [inline doc/source/conduct/code_of_conduct.rst]:
```suggestion
- Egor Panfilov <egor.v.panfilov@gmail.com>

```

**Pull request merged**